hugo/layouts/report.html

```diff
@@ -263,10 +271,17 @@ <h2 class="mb-3">Your Home's Top Characteristics</h2>
               <td><strong>Year Built</strong></td>
               <td>{{ .card.chars.char_yrblt }}</td>
             </tr>
-            <tr>
-              <td><strong>Building Square Footage</strong></td>
-              <td>{{ .card.chars.char_bldg_sf }}</td>
-            </tr>
+            {{ if .special_case_multi_card }}
```

Comment:
I tried pulling this from the global context with the `$` operator but I couldn't get it to work so now it is passed in the `card-content` template. If we think that we should be pulling it globally instead of defining in this context, I can work more on that solution

Reply:
That's OK, I think the lack of `.Params` access is indeed a limitation of the `template` approach that we're using here.